src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write code here
+const element = document.querySelector('.spider');
```

Comment:
Consider renaming the variable 'element' to 'spiderElement' for better readability.